lib/cisco_os_shim/nxapi/client.rb

```diff
@@ -18,8 +18,6 @@
 
 require 'json'
 require 'net/http'
-require_relative '../cisco_logger'
-require_relative '../client'
```

Comment:
Somewhat intentional. If you require 'cisco_os_shim/nxapi' (i.e., `lib/cisco_os_shim/nxapi.rb`) then that file auto-requires `'cisco_os_shim'`. Requiring it again in this file makes it possible to load this file in isolation rather than the entire module but I'm not sure we want to support that. Thoughts?